ui/js/values.js

```diff
@@ -11,6 +11,7 @@ treeherder.value("thPlatformNameMap", {
     "osx-10-11": "OS X 10.11",
     "windowsxp": "Windows XP",
     "windows7-32": "Windows 7",
+    "windows7-32 VM": "Windows 7 VM",
```

Comment:
How about 'windows7-32-vm' for consistency with the other entry's whitespace and capitalisation?